cpp/kiss_icp/core/Preprocessing.hpp

```diff
@@ -39,7 +42,4 @@ std::vector<Eigen::Vector3d> Preprocess(const std::vector<Eigen::Vector3d> &fram
 /// Originally introduced the calibration factor)
 std::vector<Eigen::Vector3d> CorrectKITTIScan(const std::vector<Eigen::Vector3d> &frame);
 
-/// Voxelize point cloud keeping the original coordinates
```

Comment:
Why did you change the place? I have nothing against it, just makes the diff larger

Reply:
PTSD, the order in the ```.hpp``` and ```.cpp``` was different, it drive me nuts, not it is consistent.

Reply:
All great, I'd prefer to leave style changes out of the scope of core changes; when we look back to this PR, it would be better to know the exact real change and not mixin stuff we want to fix.